<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LatexExportWidget</class>
 <widget class="QDialog" name="LatexExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>388</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0">
    <widget class="QTabWidget" name="_tab">
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Widget2">
      <attribute name="title">
       <string>Document</string>
      </attribute>
      <layout class="QHBoxLayout" name="documentLayout">
       <item>
        <layout class="QFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="groupBox">
           <property name="text">
            <string>Document style:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="latexStyleButton">
           <property name="text">
            <string>Latex style</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="wordsStyleButton">
           <property name="text">
            <string>Words style</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="groupBox">
           <property name="text">
            <string>Document type:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="fullDocButton">
           <property name="toolTip">
            <string>The document will be able to be compiled alone</string>
           </property>
           <property name="whatsThis">
            <string>The document will be generated as a full latex document since all the include will be generated before the \begin{document} and \end{document} commands.</string>
           </property>
           <property name="text">
            <string>Independent document</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="embededButton">
           <property name="toolTip">
            <string>Do not generate either the latex header or the document environment</string>
           </property>
           <property name="whatsThis">
            <string>The document will be generated as a latex document which will have to be included in a main latex document. It will allow you to generate several little files for each chapter of your document.</string>
           </property>
           <property name="text">
            <string>Document to include</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabel1_2">
           <property name="text">
            <string>Document class:</string>
           </property>
           <property name="buddy">
            <cstring>classComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KComboBox" name="classComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="TextLabel2">
           <property name="text">
            <string>Quality:</string>
           </property>
           <property name="buddy">
            <cstring>qualityComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="KComboBox" name="qualityComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Final</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Draft</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="defaultFontSizeTextZone">
           <property name="text">
            <string>Default font size:</string>
           </property>
           <property name="buddy">
            <cstring>defaultFontSize</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="defaultFontSize">
           <property name="value">
            <number>12</number>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Widget3">
      <attribute name="title">
       <string>Pictures</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Convert the pictures:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="pictureCheckBox">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Pictures directory:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KUrlRequester" name="pathPictures"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
       <item>
        <layout class="QHBoxLayout">
         <item column="0">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item column="1">
          <widget class="QLabel">
           <property name="text">
            <string>Encoding:</string>
           </property>
          </widget>
         </item>
         <item column="2">
          <widget class="KComboBox" name="encodingComboBox"/>
         </item>
         <item column="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Language</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
            <item>
             <widget class="QListWidget" name="languagesList">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="addLanguageBtn">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rmLanguageBtn">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="langUsedList">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Contents</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Export information about author:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="exportAuthor">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Add a table of content:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="tableContents">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Notes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QRadioButton" name="RadioButton7">
            <property name="text">
             <string>Don't export</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RadioButton5">
            <property name="text">
             <string>Export notes in margin notes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RadioButton6">
            <property name="text">
             <string>Export notes in comments</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1">
    <widget class="QDialogButtonBox" name="buttonBox">
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>KComboBox</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
